### DISCUSSION TITLE
Stuttering on autocompletion

### DISCUSSION THREAD

**Discussion by DeLuffyc** [dbeaver/dbeaver#20283] [Q&A]
Bought new laptop and there is stuttering when I'm texting my scripts, autocompletion from 3 to 5 sec. What could be the problem? Windows 11, i9-13900HX

**Comment by E1izabeth** [comment]
Could you try disabling this setting?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/f6da0106-7429-427f-8027-aa0f174fcfd4)
